lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ServiceAndContextConfigurationProvider.cs

```diff
@@ -70,13 +89,13 @@ internal ServiceAndContextConfigurationProvider(IPartitionedInputDataSet dataset
         /// <returns>Whether failed evaluator is master or not</returns>
         internal void RemoveEvaluatorIdFromPartitionIdProvider(string evaluatorId)
         {
-            if (!_partitionIdProvider.ContainsKey(evaluatorId))
+            if (!_partitionContextIdProvider.ContainsKey(evaluatorId))
```

Comment:
Here the purpose is not to get value but a validation checking to make sure there is no existing key there.